xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecAndroidMediaCodec.cpp

```diff
@@ -539,27 +539,12 @@ bool CDVDVideoCodecAndroidMediaCodec::Open(CDVDStreamInfo &hints, CDVDCodecOptio
           {
             switch (m_hints.dovi.dv_profile)
             {
-              case 0:
```

Comment:
Maybe group the cases and break adding a comment to explain why they are not considered? Just saves someone from incorrectly adding them again

Reply:
Good idea.